### PR TITLE
We changed all gray lines to fix them, and we added a death sequence …

### PR DESCRIPTION
**Pull request by nagel-g**
…to bad choice. Added doc string to the team_16_adv page

### CODE REVIEW

File: t04_refactored.py [hunk 1]
```diff
@@ -185,8 +189,35 @@ def team_15_adv():
...
+    direction = input("Which direction would you like to go? [left/right/forward/backward]")
+
+    if direction == "left":
+        print("You stumbled on a torch, pick it up and press the button to light it.")
+        print("You will see two paths in front of you.")
+        print("You use the light from the torch you found to guide you out "
+              "of the cave into the tulip garden full of people")
+    elif direction == "right":
+        print("You step on a fox's tail.")
+        choice = input("Would you run or stand still? [run/stay]")
```
Comment (sheggen) [inline t04_refactored.py:207]: Here you ask for `run` but below you check for `Run`

File: t04_refactored.py [hunk 2]
```diff
@@ -185,8 +189,35 @@ def team_15_adv():
...
+        if choice == "Run":
+            print("You run into two kind strangers, who lead you out of the cave.")
+        elif choice == "stay":
+            print("You get attacked by the fox, he invites his friends and they all have a nice dinner.")
+            dead = True  # bad decision, set the status as dead ( because you got killed by a fox)
+            if dead:
```
Comment (sheggen) [inline t04_refactored.py:213]: the kill_if_dead() function replaces this code.

File: t04_refactored.py [hunk 3]
```diff
@@ -328,8 +332,33 @@ def team_15_adv():
...
+            print("You run into two kind strangers, who lead you out of the cave.")
+        elif choice == "stay":
+            print("You get attacked by the fox, he invites his friends and they all have a nice dinner.")
+           dead = True
+        kill_if_dead(dead) # updated
+
+
+# TODO Make sure to add the additional check if the user makes the "bad" choice!
+
+# TODO Don't forget to check if your user is dead at the end of your chapter!
```
Comment (sheggen) [inline t04_refactored.py:361]: Remove this text.

File: t04_refactored.py [hunk 4]
```diff
@@ -583,7 +602,7 @@ def team_19_adv():
         print("You've chosen the special fruit, you've earned long-lasting prosperity!")
     else:
         print("Yummy! But not few and far between in these lands.")
-    kill_if_dead(dead)
+        kill_if_dead(dead)
```
Comment (sheggen) [inline t04_refactored.py:605]: It looks like you edited another teams function... revert these!

Comment (nagel-g) [reply t04_refactored.py:605]: We didn't touch that, it became like that after a pull from master. Shall we still edit it?

File: t04_refactored.py [hunk 5]
```diff
@@ -582,12 +582,31 @@ def team_15_adv():
 
     print()
 
-    kill_if_dead(dead)
+        kill_if_dead(dead)
```
Comment (sheggen) [inline t04_refactored.py:585]: You indented this team's code... unindent it one tab.

**Pull request merged**